config/stripe/plans.rb

```diff
@@ -0,0 +1,31 @@
+# This file contains descriptions of all your stripe plans
+
+# Example
+# Stripe::Plans::PRIMO #=> 'primo'
+
+Stripe.plan :platinum_plan do |plan|
+  # plan name as it will appear on credit card statements
+  plan.name = 'Berlingo News Premium Platinum Plan'
+
+  # amount in cents. This is 6.99
+  plan.amount = 1000000
+
+  # currency to use for the plan (default 'usd')
+  plan.currency = 'usd'
+
+  # interval must be either 'day', 'week', 'month' or 'year'
+  plan.interval = 'month'
+
+  # only bill once every three months (default 1)
+  plan.interval_count = 12
```

Comment:
logic wise if we have time at the end of the project we should change this to billing monthly when we have monthly plan intervals

Reply:
Added chore to look into this